Templates/project_collection/README.md

```diff
@@ -0,0 +1,212 @@
+# Instructions for filling out Project Collection Template
+The project and data collection template contains all the information necessary
+to add a new project and/or data collection to our inventory. There are three
+sheets in this Excel document: `Data collection project`, `Data collection`,
+and `values`. The first two sheets are to be filled out with the appropriate
+information about your project and data collection. The third sheet contains
+controlled vocabulary and should not be modified.
+
+If you have any questions, please feel free to email
+[data.curation@alleninstitute.org](mailto:data.curation@alleninstitute.org)
+so we can assist you with adding your metadata to the project inventory.
+
+
+Please refer to the following descriptions of values when filling `project_collection_template.xlsx`:
+
+## Data collection project
+This sheet contains information about your data collection project
```

Comment:
Should add a definition of what a "data collection project" is.  Possibly:
A **project** describes the intent of the work and encapsulates a set of data collections that are tightly connected. Often we have one project for each data collection, and a grant may have many projects.  A project is currently the level at which we provide author attribution, license, and grant information. Thus, if you have sets of work that require different sets of authors, you would divide the work into multiple data collections.

Reply:
Added this description to the document.